src/webpackConfigExtension.ts

```diff
@@ -27,6 +27,21 @@ export default (
     },
     resolve: {
       extensions: [".js", ".ts", ".jsx", ".tsx"],
+      // The spirit of the fallback configuration is to do the expected thing when encountering a
```

Comment:
```suggestion
      // The spirit of our fallback configuration is to do the expected thing when encountering a
```

I assume this is talking about our configuration? Use of "the" makes it unclear what you're talking about.